docs/glossary.md

```diff
@@ -0,0 +1,152 @@
+### Data dictionary
+:   A JSON file that describes the information contained in columns from a tabular data file, 
+    along with the meaning and properties (format of numerical data, unique “levels” 
+    of categorical data, etc.) of values in each column. In the context of Neurobagel, 
+    the meanings of columns and column values are encoded using terms from standardised vocabularies.
+
+### Data model
+**Used interchangeably with**: data schema
+
+:   A data model is a structure that has been designed with the 
+    purpose to represent a specific kind of information. 
+    It is made up of generic types or classes that are relevant
+    to the data model designers (for example "Research Participant"
+    and "Neuroimaging Dataset"), the properties these types can
+    have (for example "Age in years" and "Dataset name"), and the 
+    relationships that can exist between them (for example "is Part of").
+    The goal of a data model is to give information a structure
+    so that we can write programs that can consume the information.
+
+    The Neurobagel data model is designed to represent the kind
+    of information that is important to support the most relevant
+    cohort definition queries, and thus models types, properties,
+    and relationships that are important for this purpose.
+    It is not a static thing, and we constantly add new things to 
+    the data model as we support new use cases that rely on this
+    information.
+
+### Controlled term
+:   A unique identifier or code for a concept that is described in a controlled vocabulary.
+
+    A controlled term has a 
+
+        - a clear definition
```

Comment:
the formatting for these bullets and the below example looks a bit weird in the preview (it appears as a code block?)

I think outdenting this section should fix it.